### PR TITLE
React frontend and Flask backend to list all specs

### PR DESCRIPTION
**Pull request by calvinhzy**
Run guide:
./install.ps1
aazdev run

### CODE REVIEW

**Review by necusjz — APPROVED** [review]

File: src/backend/swagger/view/specs.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from ..model.specs import SwaggerSpecs
+import os
+import json
+
+swagger_path = os.environ['SWAGGER_PATH']
```
Comment (kairu-ms) [inline src/backend/swagger/view/specs.py:5]: swagger_path can be obtained from https://github.com/kairu-ms/aaz-dev-tools/blob/d847fef7269799113c5256fa8b67c4bc4e78dd0b/src/backend/utils/config.py#L3

**Comment by kairu-ms** [comment]
@calvinhzy
Is it possible to following this flask [custom-scripts doc](https://flask.palletsprojects.com/en/2.0.x/cli/#custom-scripts) to rename `flask run` by `aazdev run`?
And I think by following this [custom-commands doc](https://flask.palletsprojects.com/en/2.0.x/cli/#custom-commands), we can merge commands of aazdev/commands.py into flask.cli framework.

**Comment by kairu-ms** [comment]
@calvinhzy, Is it possible to use Typescript + React?

File: src/backend/swagger/view/specs.py [hunk 2]
```diff
@@ -2,22 +2,22 @@
 import os
 import json
 
-swagger_path = os.environ['SWAGGER_PATH']
+SWAGGER_PATH = os.environ.get("AAZ_SWAGGER_PATH", None)
```
Comment (kairu-ms) [inline src/backend/swagger/view/specs.py:5]:
```suggestion
from utils.config import SWAGGER_PATH
```

**Review by kairu-ms — COMMENTED** [review]
Is `flask_my_extension.egg-info` folder autogenerated or not? If it's autogenerated I think we need to ignore it for git?

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**